soc/arm/nuvoton_npcx/npcx9/Kconfig.defconfig.series

```diff
@@ -50,6 +50,14 @@ config TACH_NPCX
 	default y
 	depends on SENSOR
 
+config KSCAN_NPCX
+	default y
+	depends on KSCAN
```

Comment:
Delete - use the depends in kscan/Kconfig.npcx

Reply:
Done.